calculator-app/script.js

```diff
@@ -0,0 +1,127 @@
+const arr = [
+  ["C", "/", "*", "<"],
+  [7, 8, 9, "-"],
+  [4, 5, 6, "+"],
+  [1, 2, 3, "."],
+  ["(", 0, ")", "="],
+];
+
+// create container
+var app = document.createElement("div");
+app.className = "container";
+
```

Comment:
Try avoiding DOM implementation in JS for static HTML Elements in future. This is highly discouraged and the code becomes non efficient and slow. Only Iterative elements such as buttons should be created using JS.

Reply:
just tried if I would be able to make it alone with js.  Will consider the next time 🤔